tools/clang/test/HLSL/workgraph/work-graphs.hlsl

```diff
@@ -308,3 +308,24 @@ void node4_06(RWThreadNodeInputRecord<RECORD> input)
 {
   input.FinishedCrossGroupSharing(); // expected-error {{no member named 'FinishedCrossGroupSharing' in 'RWThreadNodeInputRecord<RECORD>'}}
 }
+
+struct R
+{
+  uint3 dtid: SV_DispatchThreadID;
+  uint3 gid: SV_GroupID;
+  uint gidx: SV_GroupIndex;
+  uint3 gtid: SV_GroupThreadID;
+};
+
+[Shader("node")]
+[NodeLaunch("coalescing")]
+[numthreads(4,4,4)]
+void node01(RWGroupNodeInputRecords<R> input,
+ R r,
+ uint vid : SV_VertexID ) // TODO: verify error after https://github.com/microsoft/DirectXShaderCompiler/issues/5768 got fixed.
```

Comment:
This is the right type. I don't know what #5768 has to do with this.

Reply:
VertexID should hit error about only support vertex shader in Sema.
Now we have error but not in sema, so verifier cannot check the error.